librz/core/agraph.c

```diff
@@ -4238,10 +4238,12 @@ RZ_API int rz_core_visual_graph(RzCore *core, RzAGraph *g, RzAnalysisFunction *_
 			break;
 			// Those hardcoded keys are useful only for aegi, should add subcommand of ag to set key actions
 		case '1':
-			rz_core_cmd0(core, "so;.aeg*");
+			rz_core_seek_opcode(core, 4, false);
```

Comment:
why 4?

Reply:
Oh yes, this is something I forgot to add as a WIP/to ask.
I was trying to find out the equivalent of calling `so`. When I ran that, four bytes got seeked. I just gave 4 even though I didn't realise how it's done.

I think we should pass 0 or something that's 'nothing'?

Reply:
Pass `1`. In your case it went ahead of 4 bytes because probably the instruction at core->offset was 4 bytes.